local-cli/automated-vs/solutions/wpfSolution.js

```diff
@@ -0,0 +1,37 @@
+'use-strict';
+
+const path = require('path');
+const chalk = require('chalk');
+const child_process = require('child_process');
+const spawn = child_process.spawn;
+const WindowsSolution = require('./windowsSolution');
+
+module.exports = class WPFSolution extends WindowsSolution {
+
+    //build
+    getWindowsSDKFolderPath(programFilesFolder) {
+        return path.join(programFilesFolder, 'Windows Kits');
+    }
+
+    //deploy
+    deploy(options, alternateWindowsPath) {
+        super.deploy(options);
+        this.deployToDesktop(options, alternateWindowsPath);
+    }
+
+    deployToDesktop(options, alternateWindowsPath) {
+        const defaultLaunchAppscript = `/bin/${options.arch}/${options.configuration}/${this.name}`;
```

Comment:
notably previously the wpf code only could deploy for the x86 and debug configuration, as this was was `./wpf/${appName}/bin/x86/Debug/${appName}`

Reply:
👍